pro/src/components/SignupJourneyForm/Authentication/OffererAuthenticationForm.tsx

```diff
@@ -16,9 +23,25 @@ export interface OffererAuthenticationFormValues
   publicName: string
   addressAutocomplete: string
   'search-addressAutocomplete': string
+  coords: string
+  manuallySetAddress: boolean
```

Comment:
Vu que c'est sous FF, ça ne devrait pas être comme ça ?

```suggestion
  coords?: string
  manuallySetAddress?: boolean
```